tartiflette_aiohttp/_handler.py

```diff
@@ -0,0 +1,136 @@
+import json
+import logging
+
+from aiohttp import web
+
+
+logger = logging.getLogger(__name__)
+
+
+def _format_errors(errors):
+    def format_error(_):
+        formatted_error = {
+            "type": "internal_error",
+            "message": "Server internal",
+        }
+        return formatted_error
+
+    results = []
+    for error in errors:
+        formatted_errors = format_error(error)
+        if not isinstance(formatted_errors, list):
+            formatted_errors = [formatted_errors]
+        results += formatted_errors
+    return results
+
+
+class BadRequestError(Exception):
+    pass
+
+
+def prepare_response(data):
+    headers = {}
+    #  Do things with header here if needed
+
+    return web.json_response(data, headers=headers, dumps=json.dumps)
+
+
+# pylint: disable=too-many-locals
+# reason: we're building a query with many components
+async def _handle_query(req, query, query_vars, query_name, context):
+
+    formatted_errors = None
+
+    try:
+        query_result = await req.app["gql_engine"].execute(
```

Comment:
Could we specify `tartiflette_engine` instead of  `gql_engine`?

Reply:
Everything is possible :). I'll do it right away.